docs/guides/deployment/heroku.rst

```diff
@@ -0,0 +1,99 @@
+.. _ref_guide_deployment_heroku:
+
+=========
+On Heroku
+=========
+
+In this guide we show how to deploy EdgeDB to Heroku using a Heroku PostgreSQL
+add-on as the backend.
+
+Because of Heroku's architecture EdgeDB must be deployed with a web app on
+Heroku. For this guide we will use a `todo app written in node <todo-repo_>`_.
+
+.. _todo-repo: https://github.com/edgedb/simpletodo/tree/main
+
+First copy the code, initialize a new git repo, and create a new heroku app.
+
+.. code-block:: bash
+
+   $ npx tiged 'edgedb/simpletodo#main' simpletodo-heroku
```

Comment:
I looked into this and `tiged` doesn't seem to be any more or less actively maintained than `degit`. I say we go with the name-brand, it's a proven quantity.